hugr-core/Cargo.toml

```diff
@@ -20,6 +20,7 @@ workspace = true
 extension_inference = []
 declarative = ["serde_yaml"]
 model_unstable = ["hugr-model"]
+default = ["model_unstable"]
```

Comment:
I don't think we intend to make this default?

Reply:
Ah good catch, that was just so that LSP would work for the file while I updated it.